src/sage/graphs/generic_graph.py

```diff
@@ -1900,6 +1900,23 @@ def to_dictionary(self, edge_labels=False, multiple_edges=False):
 
         return d
 
+    def _vertices_keys(self, vertices=None, *, sort=None):
```

Comment:
@dcoudert Help with naming this method would be very welcome... I noticed that there is a similarly named method `_vertices_keys` (with a very different purpose)

Reply:
Sorry, meanwhile inspiration had struck and I reworked this method as `_vertex_indices_and_keys`.